ibm/service/schematics/resource_ibm_schematics_agent.go

```diff
@@ -0,0 +1,1314 @@
+						},
+						"set_at": &schema.Schema{
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "When the User who set the state of the Object.",
+						},
+					},
+				},
+			},
+			"agent_kpi": &schema.Schema{
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
+				Computed:    true,
+				Description: "Schematics Agent key performance indicators.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"availability_indicator": &schema.Schema{
+							Type:        schema.TypeString,
+							Optional:    true,
+							Computed:    true,
+							Description: "Overall availability indicator reported by the agent.",
+						},
+						"lifecycle_indicator": &schema.Schema{
+							Type:        schema.TypeString,
+							Optional:    true,
+							Computed:    true,
+							Description: "Overall lifecycle indicator reported by the agents.",
+						},
+						"percent_usage_indicator": &schema.Schema{
+							Type:        schema.TypeString,
+							Optional:    true,
+							Computed:    true,
+							Description: "Percentage usage of the agent resources.",
+						},
+						"application_indicators": &schema.Schema{
+							Type:        schema.TypeList,
+							Optional:    true,
+							Computed:    true,
+							Description: "Agent application key performance indicators.",
+							Elem:        &schema.Schema{Type: schema.TypeMap},
+						},
+						"infra_indicators": &schema.Schema{
+							Type:        schema.TypeList,
+							Optional:    true,
+							Computed:    true,
+							Description: "Agent infrastructure key performance indicators.",
+							Elem:        &schema.Schema{Type: schema.TypeMap},
+						},
+					},
+				},
+			},
+			"agent_crn": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The agent crn, obtained from the Schematics agent deployment configuration.",
+			},
+			"created_at": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The agent creation date-time.",
+			},
+			"creation_by": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The email address of an user who created the agent.",
+			},
+			"updated_at": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The agent registration updation time.",
+			},
+			"updated_by": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Email address of user who updated the agent registration.",
+			},
+			"system_state": &schema.Schema{
+				Type:        schema.TypeList,
+				Computed:    true,
+				Description: "Computed state of the agent.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"status_code": &schema.Schema{
+							Type:        schema.TypeString,
+							Optional:    true,
```

Comment:
Inner attributes of computed field should also be computed.
Please make sure it is followed in all schema attributes

Reply:
done